src/renderer/modals/permission/permissionModal.tsx

```diff
@@ -87,32 +101,46 @@ export default class PermissionModal extends React.PureComponent<Props, State> {
 
     render() {
         return (
-            <Modal
-                bsClass='modal'
-                className='permission-modal'
-                show={Boolean(this.state.url && this.state.permission)}
-                id='requestPermissionModal'
-                enforceFocus={true}
-                onHide={() => {}}
-            >
-                <Modal.Header>
-                    <Modal.Title>{this.getModalTitle()}</Modal.Title>
-                </Modal.Header>
-                <Modal.Body>
-                    {this.getModalBody()}
-                </Modal.Body>
-                <Modal.Footer className={'remove-border'}>
-                    <div>
-                        <Button
-                            onClick={this.props.handleDeny}
-                        >{'Cancel'}</Button>
-                        <Button
-                            variant='primary'
-                            onClick={this.props.handleGrant}
-                        >{'Accept'}</Button>
-                    </div>
-                </Modal.Footer>
-            </Modal>
+            <IntlProvider>
+                <Modal
+                    bsClass='modal'
+                    className='permission-modal'
+                    show={Boolean(this.state.url && this.state.permission)}
+                    id='requestPermissionModal'
+                    enforceFocus={true}
+                    onHide={() => {}}
+                >
+                    <Modal.Header>
+                        <Modal.Title>{this.getModalTitle()}</Modal.Title>
+                    </Modal.Header>
+                    <Modal.Body>
+                        {this.getModalBody()}
+                    </Modal.Body>
+                    <Modal.Footer className={'remove-border'}>
+                        <div>
+                            <Button onClick={this.props.handleDeny}>
+                                <FormattedMessage
+                                    id='label.cancel'
+                                    defaultMessage='Cancel'
+                                />
+                            </Button>
+                            <Button
+                                variant='primary'
+                                onClick={this.props.handleGrant}
+                            >
+                                <FormattedMessage
+                                    id='label.accept'
+                                    defaultMessage='Accept'
+                                />
+                            </Button>
+                        </div>
+                    </Modal.Footer>
+                </Modal>
+            </IntlProvider>
         );
     }
 }
+
+t('common.permissions.canBasicAuth');
```

Comment:
Usually it's better to keep the `t` function with the code that uses it so that the translation string doesn't get forgotten about or out of sync. We've had cases before where we removed some text from the web app, but the `t` function still existed so it was still showing up as something that needed to be translated